client/lib/domains/registration/availability-messages.js

```diff
@@ -184,7 +184,7 @@ function getAvailabilityNotice( domain, error, errorData ) {
 			}
 
 			message = translate(
-				'Domains registration is unavailable at this time. Please select a free WordPress.com ' +
+				'Domain registration is unavailable at this time. Please select a free WordPress.com ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Domains registration is unavailable at this time. Please select a free WordPress.com domain or check back %(maintenanceEnd)s.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).